lib/seahorn/BvOpSem2.cc

```diff
@@ -553,6 +555,32 @@ class OpSemVisitor : public InstVisitor<OpSemVisitor>, OpSemVisitorBase {
     setValue(*CS.getInstruction(), res);
   }
 
+  void visitAssertIf(CallSite CS) {
+    Expr ante = lookup(*CS.getArgument(0));
+    Expr conseq = lookup(*CS.getArgument(1));
+    m_ctx.m_z3_solver->reset();
```

Comment:
as a rule, we don't access fields of objects directly. Use a getter method.

Reply:
Agreed. That is why this is a draft :). Fixed it now.